typing_tests/pine-options.ts

```diff
@@ -1,545 +1,586 @@
+				$expand: {
+					device_environment_variable: {
+						$filter: {
+							name: 'name',
+							value: null,
+							// @ts-expect-error test case
+							asdf: 'asdf',
 						},
 					},
 				},
 			},
-			'depends_on__application',
-		],
-	};
+		},
+		'organization',
+	],
+};
 
 // this check that even though the unknown$any$filterPropInsideArray$expandPlural case doesn't work
 // it will at least not silence other errors
-export const unknown$filterPropInsideNested$expandWithUnknown$any$filterProp: BalenaSdk.PineOptions<BalenaSdk.Application> =
-	{
-		$expand: [
-			{
-				owns__device: {
-					$filter: {
-						device_name: null,
-						note: 'note',
-						device_environment_variable: {
-							$any: {
-								$alias: 'dev',
-								$expr: {
-									dev: {
-										name: 'name',
-										// @ts-expect-error test case
-										asdf: 'asdf',
-									},
+export const unknown$filterPropInsideNested$expandWithUnknown$any$filterProp: BalenaSdk.Pine.ODataOptions<
+	BalenaSdk.Application['Read']
+> = {
+	$expand: [
+		{
+			owns__device: {
+				$filter: {
+					device_name: null,
+					note: 'note',
+					device_environment_variable: {
+						$any: {
+							$alias: 'dev',
+							$expr: {
+								dev: {
+									name: 'name',
+									// TODO: this should error
+									asdf: 'asdf',
 								},
 							},
 						},
 					},
-					$expand: {
-						device_environment_variable: {
-							$filter: {
-								name: 'name',
-								value: null,
-								// @ts-expect-error test case
-								asdf: 'asdf',
-							},
+				},
+				$expand: {
+					device_environment_variable: {
+						$filter: {
+							name: 'name',
+							value: null,
+							// @ts-expect-error test case
+							asdf: 'asdf',
 						},
 					},
 				},
 			},
-			'depends_on__application',
-		],
-	};
+		},
+		'organization',
+	],
+};
 
-export const string$OrderbyWoDirection: BalenaSdk.PineOptions<BalenaSdk.Application> =
-	{
-		// @ts-expect-error test case
-		$orderby: 'id',
-	};
+export const string$OrderbyWoDirection: BalenaSdk.Pine.ODataOptions<
+	BalenaSdk.Application['Read']
+> = {
+	// TODO: this should error
+	$orderby: 'id',
+};
 
-export const string$OrderbyWoPropPrefix: BalenaSdk.PineOptions<BalenaSdk.Application> =
-	{
-		// @ts-expect-error test case
-		$orderby: 'asc',
-	};
+export const string$OrderbyWoPropPrefix: BalenaSdk.Pine.ODataOptions<
+	BalenaSdk.Application['Read']
+> = {
+	// @ts-expect-error test case
+	$orderby: 'asc',
+};
 
-export const string$OrderbyInvalidDirection: BalenaSdk.PineOptions<BalenaSdk.Application> =
-	{
-		// @ts-expect-error test case
-		$orderby: { id: 'ascending' },
-	};
+export const string$OrderbyInvalidDirection: BalenaSdk.Pine.ODataOptions<
+	BalenaSdk.Application['Read']
+> = {
+	// @ts-expect-error test case
+	$orderby: 'id ascending',
+};
 
-export const string$OrderbyInvalidDirectionCase: BalenaSdk.PineOptions<BalenaSdk.Application> =
-	{
-		// @ts-expect-error test case
-		$orderby: { id: 'Asc' },
-	};
```

Comment:
Should we also add these two back, to make sure that the `dir` is checked?
If of course they are not failing, we can add then w/ a todo instead of a ts-expect-error
```
export const object$OrderbyInvalidDirection: BalenaSdk.PineOptions<BalenaSdk.Application> =
	{
		// @ts-expect-error test case
		$orderby: { id: 'ascending' },
	};


export const object$OrderbyInvalidDirectionCase: BalenaSdk.PineOptions<BalenaSdk.Application> =
	{
		// @ts-expect-error test case
		$orderby: { id: 'Asc' },
	};
```

Reply:
They are there... it is just github diff missleading you